docs/resources/vault_secrets_rotating_secret.md

```diff
@@ -82,6 +94,7 @@ resource "hcp_vault_secrets_rotating_secret" "example_confluent" {
 ### Optional
 
 - `aws_access_keys` (Attributes) AWS configuration to manage the access key rotation for the given IAM user. Required if `secret_provider` is `aws`. (see [below for nested schema](#nestedatt--aws_access_keys))
+- `azure_application_password_params` (Attributes) Azure configuration to manage the application password rotation for the given application. Required if `secret_provider` is `confluent`. (see [below for nested schema](#nestedatt--azure_application_password_params))
```

Comment:
```suggestion
- `azure_application_password` (Attributes) Azure configuration to manage the application password rotation for the given application. Required if `secret_provider` is `azure`. (see [below for nested schema](#nestedatt--azure_application_password_params))
```

The param suffix feels redundant and the other credential types did not add it. There is a Confluent copy-paste leftover.

Reply:
Whoops I think this is the generated code. Nvm the suggestion but the comment are relevant to the resource schema.

Reply:
Thanks for noticing this. Updated.